test/activate_set.jl

```diff
@@ -1,12 +1,18 @@
 @testset "activate_set.jl" begin
     @testset "activate [extras]" begin
         orig_project_toml_path = Base.active_project()
+        push!(LOAD_PATH, mktempdir())  # put something weird in LOAD_PATH for testing
+        orig_load_path = Base.LOAD_PATH
         try
             TestEnv.activate("ChainRulesCore")
             new_project_toml_path = Base.active_project()
             @test new_project_toml_path != orig_project_toml_path
+            @test orig_load_path ⊆ Base.LOAD_PATH
```

Comment:
this could be
```suggestion
            @test orig_load_path == Base.LOAD_PATH
```
but idk if that is a promise i want to make.
Maybe for now it is and then can relax it later?

Reply:
i think `==`  makes sense, but idk i've never really worked much with Pkg internals